src/test/java/uk/gov/ons/census/casesvc/service/RefusalServiceTest.java

```diff
@@ -70,4 +72,37 @@ public void testRefusalForCase() {
             eq(messageTimestamp));
     verifyNoMoreInteractions(eventLogger);
   }
+
+  @Test(expected = RuntimeException.class)
+  public void testRefusalNotFromFieldForEstabAddressLevelCaseThrowsException() {
+    // GIVEN
+    ResponseManagementEvent managementEvent = getTestResponseManagementRefusalEvent();
+    managementEvent.getEvent().setChannel("NOT FROM FIELD");
+    managementEvent.getPayload().getRefusal().getCollectionCase().setId(TEST_CASE_ID.toString());
+
+    Case testCase = getRandomCase();
+    testCase.setAddressLevel(ESTAB_ADDRESS_LEVEL);
+    OffsetDateTime messageTimestamp = OffsetDateTime.now();
+
+    when(caseService.getCaseByCaseId(TEST_CASE_ID)).thenReturn(testCase);
+    String expectedErrorMessage =
+        String.format(
+            "Refusal received for Estab level case ID '%s' from channel '%s'. "
+                + "This type of refusal should ONLY come from Field",
+            testCase.getCaseId(), managementEvent.getEvent().getChannel());
+
+    // WHEN
+    try {
+      underTest.processRefusal(managementEvent, messageTimestamp);
+    } catch (RuntimeException expectedException) {
+      // THEN
+      assertThat(expectedException.getMessage()).isEqualTo(expectedErrorMessage);
+
+      verify(caseService, times(1)).getCaseByCaseId(any(UUID.class));
+      verifyNoMoreInteractions(caseService);
+      verifyZeroInteractions(eventLogger);
+
+      throw expectedException;
```

Comment:
this kind of re-throw is dangerous, because ANY kind of RuntimeException could get thrown from almost anywhere in the code and the test would pass. I'm more in favour of having a boolean that marks the expected exception got thrown in this kind of scenario. Happy enough to approve though